docs/source/core_developers_guide/environment_setup.rst

```diff
@@ -144,3 +174,182 @@ install_sphinx
+
+Open up a terminal and run the following:
+
+.. code-block:: console
+
+   % cd $HOME
+   % mkdir project
+   % cd project
+   % git clone https://github.com/hyperledger/sawtooth-core.git
+
+.. note::
+
+  On a Windows environment, the suggested version of the last command
+  above is:
+
+  .. code-block:: console
+
+      C:\> git clone https://github.com/hyperledger/sawtooth-core.git
+      --config core.autocrlf=false
+
+
+Step Two: Configure Proxy (Optional)
+====================================
+
+If you are behind a network proxy, follow these steps before continuing:
+
+1. Set the following environment variables:
+
+  * http_proxy
+  * https_proxy
+
+If you are using the Bash shell, run the following commands:
+
+.. warning::
+
+  The example URLs and port numbers used below are examples only.
+  Please substitute the actual URL, with actual port numbers, used
+  in your environment. Contact your network administrator for the
+  information if necessary.
+
+.. code-block:: console
+
+  % export http_proxy=http://example-proxy-server.com:3128
+  % export https_proxy=https://example-proxy-server.com:3129
+
+If you are using Windows, run the following commands:
+
+.. code-block:: console
+
+  % set http_proxy=http://example-proxy-server.com:3128
+  % set https_proxy=https://example-proxy-server.com:3129
+
+
+2. Install the vagrant-proxyconf plugin:
+
+.. code-block:: console
+
+  % cd sawtooth-core/tools
+  % vagrant plugin install vagrant-proxyconf
+
+
+Step Three: Start VM
+====================
+
+In order to start the vagrant VM, run:
+
+.. code-block:: console
+
+  % cd sawtooth-core/tools
+  % vagrant up
+
+.. note::
+
+   We have encountered an intermittent problem on Windows hosts which
+   presents as an 'Operation not permitted' error in the vagrant startup
+   output. If you encounter this error, perform a 'vagrant destroy' and
+   then run 'vagrant up' again.
+
+Downloading the Vagrant box file, booting the VM, and running through
+the bootstrap scripts will take several minutes.
+
+Once the 'vagrant up' command has finished executing, run:
+
+.. code-block:: console
+
+  % vagrant ssh
+
+By default, Vagrant sets up ssh keys so that users can log into the VM
+without setting up additional accounts or credentials. The logged in user,
+vagrant (uid 1000), also has permissions to execute sudo with no password
+required. Any number of `vagrant ssh` sessions can be established from the
+host.
+
+.. note::
+
+   Occasionally, the configuration files used to create the vagrant
+   environment change and this can result in vagrant asking for a password when
+   doing `vagrant up` or `vagrant ssh`. This is usually a sign that your
+   environment is out of date. If this happens, please follow the instructions
+   below to reset your environment. This is especially common when switching
+   between versions of Sawtooth Lake, eg. 0.7 -> 0.8.
+
+
+
+Resetting The Environment (Optional)
```

Comment:
Removed "Step" verbiage. The order should make it fairly obvious.
changed "Resetting" to "Reset."